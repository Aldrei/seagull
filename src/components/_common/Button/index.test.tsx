import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import { ButtonCommon } from './index'
 
describe('Common components', () => {
  it('renders the ButtonCommon', () => {
    const el = render(<ButtonCommon text='test' />)
    expect(el.baseElement).toMatchSnapshot()
  })
})
