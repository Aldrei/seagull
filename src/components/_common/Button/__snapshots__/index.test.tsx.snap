// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Common components renders the ButtonCommon 1`] = `
<body>
  <div>
    <button
      class="sc-44c70c92-0 jlqeCN button-common undefined"
    >
      test
    </button>
  </div>
</body>
`;
